homeassistant/components/binary_sensor/netatmo.py

```diff
@@ -49,48 +62,68 @@ def setup_platform(hass, config, add_devices, discovery_info=None):
     netatmo = get_component('netatmo')
     home = config.get(CONF_HOME, None)
     timeout = config.get(CONF_TIMEOUT, 15)
+    offset = config.get(CONF_OFFSET, 90)
```

Comment:
I feel that we could do something cleaner here, as this two parameter are doing almost the same thing, and might create confusion for the end-user. but it should OK for the time being.